src/rpc/client.cpp

```diff
@@ -60,12 +60,12 @@ static const CRPCConvertParam vRPCConvertParams[] =
         {"getbalance", 3},
         {"getshieldedbalance", 1},
         {"getshieldedbalance", 2},
-        { "raw_shielded_sendmany", 1},
-        { "raw_shielded_sendmany", 2},
-        { "raw_shielded_sendmany", 3},
-        { "shielded_sendmany", 1},
-        { "shielded_sendmany", 2},
-        { "shielded_sendmany", 3},
+        { "rawshieldedsendmany", 1},
+        { "rawshieldedsendmany", 2},
+        { "rawshieldedsendmany", 3},
+        { "shieldedsendmany", 1},
+        { "shieldedsendmany", 2},
+        { "shieldedsendmany", 3},
```

Comment:
since we are at it, let's remove the extra space after `{`.

Reply:
Yeah this was something that was done a couple years ago in a blanket `clang-format` application over the entire repository. I was against it then and still am. So, to bring us back to upstream standards for readability (which is to have a whitespace buffer inside the braces for this list), i've added a second commit to this PR.

Doing this will also avoid some backport conflicts (a trivial issue, but also trivial effort to avoid)